python_modules/libraries/dagster-pagerduty/dagster_pagerduty/resources.py

```diff
@@ -93,7 +91,7 @@ def EventV2_create(
 
             component {string} -- The part or component of the affected system that is broken.
 
-                                  Examples:
+        Examples:
```

Comment:
Hmmm this doesn't not seem like a win to me in this case. These examples are specific to this parameter

Reply:
The docstring was not google style, I've now converted it.